1-WebApp-OIDC/1-5-B2C/Views/Home/Index.cshtml

```diff
@@ -3,10 +3,10 @@
 }
 
 <h1>
-    ASP.NET Core web app signing-in users in your organization
+    ASP.NET Core web app signing-in users in B2C
 </h1>
 <p>
-    This sample shows how to build a .NET Core 2.2 MVC Web app that uses OpenID Connect to sign in users in your organization. It leverages the ASP.NET Core OpenID Connect middleware.
+    This sample shows how to build a .NET Core 2.2 MVC Web app that uses OpenID Connect to sign in users in B2C. It leverages the ASP.NET Core OpenID Connect middleware.
```

Comment:
>B2C [](start = 108, length = 3)

they are signing in w/a variety of IdPs...so maybe reword this somehow to include that?